tools/crm_diff.c

```diff
@@ -145,61 +145,17 @@ log_patch_cib_versions(xmlNode *patch)
     int add[] = { 0, 0, 0 };
     int del[] = { 0, 0, 0 };
 
-    const char *fmt = NULL;
     const char *digest = NULL;
 
     xml_patch_versions(patch, add, del);
-    fmt = crm_element_value(patch, "format");
```

Comment:
by contrast, this will be fine once we do drop the ability to generate v1 patches